rllib/tuned_examples/ppo/multi_agent_footsies_ppo.py

```diff
@@ -0,0 +1,262 @@
+#    2. num_env_steps_sampled_lifetime (via "default_timesteps")
+# ...values very high to make sure that the test passes by adding all required policies to the mix, not by hitting the iteration limit.
+# Our main stopping criterion is "target_mix_size" (see an argument below).
+parser = add_rllib_example_script_args(
+    default_iters=500,
+    default_timesteps=5_000_000,
+)
+
+parser.add_argument(
+    "--train-start-port",
+    type=int,
+    default=45001,
+    help="First port number for the Footsies training environment server (default: 45001). Each server gets its own port.",
+)
+parser.add_argument(
+    "--eval-start-port",
+    type=int,
+    default=55001,
+    help="First port number for the Footsies evaluation environment server (default: 55001) Each server gets its own port.",
+)
+parser.add_argument(
+    "--binary-download-dir",
+    type=Path,
+    default="/tmp/ray/binaries/footsies",
+    help="Directory to download Footsies binaries (default: /tmp/ray/binaries/footsies)",
+)
+parser.add_argument(
+    "--binary-extract-dir",
+    type=Path,
+    default="/tmp/ray/binaries/footsies",
+    help="Directory to extract Footsies binaries (default: /tmp/ray/binaries/footsies)",
+)
+parser.add_argument(
+    "--binary-to-download",
+    type=str,
+    choices=["linux_server", "linux_windowed", "mac_headless", "mac_windowed"],
+    default="linux_server",
+    help="Target binary for Footsies environment (default: linux_server). Linux and Mac machines are supported. "
+    "'linux_server' and 'mac_headless' choices are the default options for the training. Game will run in the batchmode, without initializing the graphics. "
+    "'linux_windowed' and 'mac_windowed' choices are for the local run only, because "
+    "game will be rendered in the OS window. To use this option effectively, set up: "
+    "--no-tune --num-env-runners 0 --evaluation-num-env-runners 0",
+)
+parser.add_argument(
+    "--win-rate-threshold",
+    type=float,
+    default=0.9,
+    help="The main policy should have at least 'win-rate-threshold' win rate against the "
+    "other policy to advance to the next level. Moving to the next level "
+    "means adding a new policy to the mix.",
+)
+parser.add_argument(
+    "--target-mix-size",
+    type=int,
+    default=8,
+    help="Target number of policies (RLModules) in the mix to consider the test passed. "
+    "The initial mix size is 2: 'main policy' vs. 'other'. "
+    "`--target-mix-size=8` means that 6 new policies will be added to the mix. "
+    "Whether to add new policy is decided by checking the '--win-rate-threshold' condition. ",
+)
+parser.add_argument(
+    "--rollout-fragment-length",
+    type=int,
+    default=256,
+    help="The length of each rollout fragment to be collected by the EnvRunners when sampling.",
+)
+
+
+if __name__ == "__main__":
+    main_policy = "lstm"
+    args = parser.parse_args()
+
+    register_env(name="FootsiesEnv", env_creator=env_creator)
+
+    config = (
```

Comment:
We shoujld always move the config itself out of the `__main__` block. Then it can be imported by other scripts. If you have to move some other variables (e.g. args) with it, that'd also be ok.